PrismaDB-QueryAST/DML/Expressions/AllColumns.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.Data;
+
+namespace PrismaDB.QueryAST.DML
+{
+    public class AllColumns : Expression
+    {
+        public TableRef Table;
+
+        public AllColumns(string tableName)
+        {
+            setValue(tableName);
+        }
+
+        public AllColumns()
+            : this("")
+        { }
+
+        public AllColumns(TableRef table)
+            : this()
+        {
+            Table = table.Clone();
+        }
+
+        public override void setValue(params object[] value)
+        {
+            Parent = null;
+            Table = new TableRef((string)value[0]);
+        }
+
+        public override object Clone()
+        {
+            var clone = new AllColumns(Table);
+
+            return clone;
+        }
+
+        public override object Eval(DataRow r)
+        {
+            return r[ToString()];
```

Comment:
I think this expression should throw an exception on Eval as it shouldn’t appear in Where

Reply:
ok

Reply:
done